releases/cloud/point-2024-10-01-release-note.md

```diff
@@ -0,0 +1,30 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.4.64.5
+---
+
+# October 01, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
+
+
+## Improvements & Bug Fixes
+
+* **Sensitivity Labels** can now be marked as sensitive by administrators in the Settings of Syskit Point. 
+  * The Sensitivity Label with the highest priority rating is automatically set as sensitive.
+  * Administrators can find the list of Sensitivity Labels in settings and define which should be registered as sensitive.
+
```

Comment:
fali mi info čemu to služi

Reply:
u settingsima imamo odgovor: Content marked with such label(s) will be shown in reports that display sensitive content.